.github/workflows/kapua-ci.yaml

```diff
@@ -31,710 +30,283 @@ jobs:
-      - run: mvn -B ${BUILD_OPTS} -Pdocker -DskipTests clean install
-      - uses: nick-fields/retry@v2.8.1
-        with:
-          timeout_minutes: 45
-          retry_on: error
-          max_attempts: 1
-          command: ./ci-output.sh mvn -B ${BUILD_OPTS} ${CONFIG_OVERRIDES} -Dgroups='!org.eclipse.kapua.qa.markers.junit.JUnitTests' -Dcucumber.filter.tags="@triggerService" verify
-      - run: bash <(curl -s https://codecov.io/bash)
-  test-triggerServiceIntegrationBase:
-    needs: build
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-java@v3
-        with:
-          distribution: 'zulu'
-          java-version: 11
-          cache: 'maven'
-      - uses: actions/cache@v3
-        with:
-          path: ~/.m2/repository
-          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
-          restore-keys: |
-            ${{ runner.os }}-maven-
-      - run: echo "127.0.0.1       message-broker" | sudo tee -a /etc/hosts
-      - run: echo "127.0.0.1       job-engine" | sudo tee -a /etc/hosts
-      - run: mvn -B ${BUILD_OPTS} -Pdocker -DskipTests clean install
-      - uses: nick-fields/retry@v2.8.1
-        with:
-          timeout_minutes: 45
-          retry_on: error
-          max_attempts: 1
-          command: ./ci-output.sh mvn -B ${BUILD_OPTS} ${CONFIG_OVERRIDES} -Dgroups='!org.eclipse.kapua.qa.markers.junit.JUnitTests' -Dcucumber.filter.tags="@triggerServiceIntegrationBase" verify
-      - run: bash <(curl -s https://codecov.io/bash)
-  test-triggerServiceIntegration:
-    needs: build
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-java@v3
-        with:
-          distribution: 'zulu'
-          java-version: 11
-          cache: 'maven'
-      - uses: actions/cache@v3
-        with:
-          path: ~/.m2/repository
-          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
-          restore-keys: |
-            ${{ runner.os }}-maven-
-      - run: echo "127.0.0.1       message-broker" | sudo tee -a /etc/hosts
-      - run: echo "127.0.0.1       job-engine" | sudo tee -a /etc/hosts
-      - run: mvn -B ${BUILD_OPTS} -Pdocker -DskipTests clean install
-      - uses: nick-fields/retry@v2.8.1
-        with:
-          timeout_minutes: 45
-          retry_on: error
-          max_attempts: 1
-          command: ./ci-output.sh mvn -B ${BUILD_OPTS} ${CONFIG_OVERRIDES} -Dgroups='!org.eclipse.kapua.qa.markers.junit.JUnitTests' -Dcucumber.filter.tags="@triggerServiceIntegration" verify
-      - run: bash <(curl -s https://codecov.io/bash)
+      - name: Clones Kapua repo inside the runner
+        uses: actions/checkout@v3
+      - uses: ./.github/actions/runTestsTaggedAs
+        with:
+          tag: '@jobsIntegration'
+          needs-docker-images: 'true'
+  #the next 3 tests-suites have been disabled because in the code-base I can't find feature files tagged in this way
```

Comment:
As a general rule: do not comment, delete. If we need to restore this in the future, we can retrive the old code from previous commits.